pySDC/core/Controller.py

```diff
@@ -37,7 +39,7 @@ class controller(object):
     Base abstract controller class
     """
 
-    def __init__(self, controller_params):
+    def __init__(self, controller_params, description):
```

Comment:
Hm.. why not have this as part of the `controller_params`? Or even dedicated `convergence_params`?

Reply:
I guess my problem with this is that the description goes into the steps and gets unwrapped there. It should not define the controller, esp. given that we have `controller_params`.